src/api/protectedApiClient.ts

```diff
@@ -29,15 +29,15 @@ const changePassword = (request: {
     .catch((e) => e);
 };
 
-const deleteUser = (request: { password: string }): Promise<void> => {
-  return AppAxiosInstance.post(ProtectedApiClientRoutes.DELETE_USER, request)
-    .then((r) => r.data)
-    .catch((e) => e);
+const deactivateAccount = (): Promise<void> => {
+  return AppAxiosInstance.delete(ProtectedApiClientRoutes.USER)
+    .then((res) => res)
```

Comment:
For some reason, even if you return something in the `then` statement, it doesn't complain that it's mismatching the return type of `Promise<void>`. Right now you're returning the result, so let's make it consistent with the return type.

```suggestion
    .then(() => {})
```